src/reducers/alert.ts

```diff
@@ -5,18 +5,22 @@ interface Options {
   alertType?: string
   showCloseLink?: boolean
   value: string | null
+  isInline?: boolean
 }
 
 /** Set an alert with an optional close link. */
-const alert = (state: State, { alertType, showCloseLink, value }: Options) => ({
-  ...state,
-  alert: value
-    ? {
-        alertType,
-        showCloseLink: showCloseLink !== false,
-        value,
-      }
-    : null,
-})
+const alert = (state: State, { alertType, showCloseLink, value, isInline = false }: Options) => {
```

Comment:
You can omit `= false` and let it default to undefined.

Reply:
Sure